visual-inspection/inspection-backend/inspection/explainer/explainers/config.py

```diff
@@ -0,0 +1,28 @@
+from pydantic import BaseModel
```

Comment:
Die Konfiguration sollte jeweils pro Explainer eigenständig sein. Wir haben bei LIME und VisuaLIME verschiedene Parameter, VisuaLIME hat in jedem Fall deutlich mehr Optionen. Und wenn wir noch weitere Explainer einbauen, die vielleicht gar nichts mit LIME als Algorithmus zu tun haben, dann haben die wiederum eine eigene Art der Konfiguration.

Es ist auch kein Problem, wenn das erstmal eine 1:1-Kopie ist, im Laufe der Zeit ändert sich das dann.